chaos-test/chaostest/requirements.txt

```diff
@@ -2,7 +2,7 @@ chaostoolkit-lib>=0.13.0
 requests
 coloredlogs
 jsonpath-rw-ext==1.2.2
-chaostoolkit-kubernetes
+chaostoolkit-kubernetes>=0.25.1
```

Comment:
The default will be the latest, so if it is not specified then it will pick the latest only, any reason we want particular version or we are good with the latest ?